libs/langchain-google-common/src/chat_models.ts

```diff
@@ -74,7 +91,7 @@ export abstract class ChatGoogleBase<AuthOptions>
 
   lc_serializable = true;
 
-  model = "gemini-pro";
+  modelName = "gemini-pro";
```

Comment:
Is this not breaking?

Reply:
No worries if so, but why change

Reply:
Technically yes, but these were never documented before, and only have ~250 downloads/week.

Removed to keep consistency